docs/docs-content/troubleshooting/generate-har-files.md

```diff
@@ -0,0 +1,155 @@
+    5. On the page, reproduce the issues you've encountered.
+    6. In the **Network** tab menu, select **Export**.
+  </TabItem>
+  <TabItem value="firefox" label="Firefox">
+    1. Open the page with issues in Firefox.
+    2. On your keyboard, open Developer Tools:
+       - For Windows and Linux, press **F12** or **Ctrl + Shift + I**.
+       - For Mac, press **Fn + F12** or **Cmd + Option + I**.
+
+       For alternative methods to open Developer Tools, refer to [Firefox DevTools User Docs](https://firefox-source-docs.mozilla.org/devtools-user/).
+    3. In Developer Tools, select the **Network** tab and, in the **Network** tab menu:
+       - Select <FontAwesomeIcon icon={faTrashCan} /> to clear the current network log.
+       - Toggle <FontAwesomeIcon icon={faPlay} color={"#007AFF"} /> to <FontAwesomeIcon icon={faPause} /> to start recording network activity.
+       - Select <FontAwesomeIcon icon={faGear} /> > **Persist Logs**. 
+
+       ![](/assets/docs/images/troubleshooting/generate-har-files/firefox-devtools.png)
+    4. On the page, reproduce the issues you've encountered.
+    5. In the network log, right-click any item and select **Save All As HAR**.
+  </TabItem>
+</Tabs>
+
+## Sanitize HAR Files
+
+1. Open the generated HAR file in a text editor.
+2. Carefully examine the file contents and either remove or redact sensitive information.
+   
+   You can use the following keywords for reference:
+   
+    :::caution
+    This list is not exhaustive. You should also check for data that is considered sensitive or confidential within your organization.
+    :::
+
+   - **state**
+   - **shdf**
+   - **usg**
+   - **password**
+   - **code**
+   - **code_verifier**
+   - **client_secret**
+   - **token**
+   - **Access_token**
+   - **refresh_token**
+   - **authenticity_token**
+   - **Id_token**
+   - **SAMLResponse**
+   - **SAML Request**
+   - **appID**
+   - **challenge**
+   - **facetID**
+   - **assertion**
+   - **fcParams**
+   - **serverData**
+   - **Authorization**
+   - **auth**
+   - **key**
+   - **pem**
+   - **rsa**
+   - **dsa**
+   - **ecdsa**
+   - **signature**
+   - **passkey**
+
+Consider the following examples of how to approach redacting passwords, tokens, and cookies.
+
+### Password Example
+
+```json
+  "postData": {
+    "mimeType": "application/json",
+    // highlight-next-line
+    "text": "{\"emailId\":\"REDACTED\",\"password\":\"REDACTED\",\"org\":\"spectro-cloud\"}"
+  }
+```
+
+### Token Example
+
+```json
+  "queryString": [
+    {
+      "name": "access_token",
+    // highlight-next-line
+      "value": "REDACTED"
+    }
+  ],
+```
+
+### Cookie Example
+
+```json
+  "cookies": [
+    {
+      "name": "__stripe_mid",
+      // highlight-next-line
+      "value": "REDACTED",
+      "path": "/",
+      "domain": "console.spectrocloud.com",
+      "expires": "2025-01-10T20:51:03.000Z",
+      "httpOnly": false,
+      "secure": true,
+      "sameSite": "Strict"
+    },
+  ]
+```
```

Comment:
I think this still leaves me hanging a little bit, what happens after I sanitize the file? I imagine it's sending to Spectro Cloud support, it may be beneficial to provide that context here.